angular.json

```diff
@@ -40,5 +40,8 @@
       }
     }
   },
-  "defaultProject": "angular-archwizard"
-}
+  "defaultProject": "angular-archwizard",
+  "cli": {
+    "analytics": false
+  }
```

Comment:
What does this option do?

Reply:
ops, my bad on that one. Not needed at all